lib/src/layer/marker_layer.dart

```diff
@@ -8,6 +8,10 @@ import 'package:latlong2/latlong.dart';
 class MarkerLayerOptions extends LayerOptions {
   final List<Marker> markers;
 
+  /// Do we want to cache projections for performance, caching can
+  /// introduce inaccuracies if the cache is out of date
```

Comment:
Consider using more 'direct' documentation?

```suggestion
  /// Toggle marker position caching. Enabling will improve performance, but may introduce
  /// errors when adding/removing markers. Default is enabled (`true`).
```